events/sick_man_events.txt

```diff
@@ -0,0 +1,664 @@
+# start event
+sick_man.1 = {
+	type = country_event
+	placement = root
+	title = sick_man.1.t
+	desc = sick_man.1.d
+	flavor = sick_man.1.f
+	duration = 3
+	event_image = {
+		video = "unspecific_factory_closed"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	icon = "gfx/interface/icons/event_icons/event_skull.dds"
+	trigger = {
+	}
+	immediate = {
+		set_variable = {
+			name = sick_man_var
+			value = 0
+		}
+		set_variable = {
+			name = sick_man_separatist_var
+			value = 0
+		}
+		add_journal_entry = {
+			type = je_sick_man_main
+		}
+	}
+	option = {
+		name = sick_man.1.a
+		add_journal_entry = {
+			type = je_sick_man_economy
+		}
+		add_journal_entry = {
+			type = je_sick_man_army
+		}
+		add_journal_entry = {
+			type = je_sick_man_navy
+		}
+		add_journal_entry = {
+			type = je_sick_man_bureaucracy
+		}
+		show_as_tooltip = {
+			add_modifier = {
+				name = sick_man_of_europe
+				months = -1
+			}
+			add_modifier = {
+				name = outmoded_bureaucracy
+				months = -1
+			}
+			add_modifier = {
+				name = cizye_tax
+				months = -1
+			}
+			add_modifier = {
+				name = stagnant_economy
+				months = -1
+			}
+			add_modifier = {
+				name = janissary_legacy
+				months = -1
+			}
+			add_modifier = {
+				name = rampant_piracy
+				months = -1
+			}
+			add_modifier = {
+				name = kurdish_question
+				months = -1
+			}
+			add_modifier = {
+				name = balkan_disarray
+				months = -1
+			}
+			add_modifier = {
+				name = imbalanced_education
+				months = -1
+			}
+			add_modifier = {
+				name = internal_strife
+				months = -1
+			}
+		}
+	}
+}
+
+# Liberal Success event
+sick_man.2 = {
+	type = country_event
+	placement = root
+	title = sick_man.2.t
+	desc = sick_man.2.desc
+	flavor = sick_man.2.f
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+	duration = 3
+	event_image = {
+		video = "europenorthamerica_london_center"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	trigger = {
+	}
+	immediate = {
+		set_variable = sick_man_complete
+		remove_modifier = sick_man_of_europe
+		remove_modifier = outmoded_bureaucracy
+		remove_modifier = cizye_tax
+		remove_modifier = stagnant_economy
+		remove_modifier = janissary_legacy
+		remove_modifier = rampant_piracy
+		remove_modifier = kurdish_question
+		remove_modifier = balkan_disarray
+		remove_modifier = imbalanced_education
+		remove_modifier = internal_strife
+	}
+	option = {
+		name = sick_man.2.a
+		default_option = yes
+		add_loyalists = {
+			value = 0.20
+			strata = middle
+		}
+		add_loyalists = {
+			value = 0.10
+			strata = rich
+		}
+		add_modifier = {
+			name = sick_man_liberal_western_happy
+			months = 120
+			is_decaying = yes
+		}
+		add_modifier = {
+			name = sick_man_liberal_western_education
+		}
+		change_relations = {
+			country = c:FRA
+			value = 30
+		}
+		change_relations = {
+			country = c:GBR
+			value = 30
+		}
+		change_relations = {
+			country = c:PRU
+			value = 30
+		}
```

Comment:
How about NGF/Germany?

Reply:
Would that be an or statement? Or an and?

Reply:
For the sake of readability, I'd do an `if` statement, with an `exists = c:PRU` in the `limit`, for example. Then put the `change_relations` for the specified country in there.

Reply:
can't they just use  change_relations ?= {

Reply:
After some testing I am struggling to properly fulfill this without errors.

Reply:
I can help out :)